react/features/participants-pane/components/web/styled.js

```diff
@@ -190,12 +190,6 @@ export const Heading = styled.div`
   margin: 8px 0 ${props => props.theme.panePadding}px;
 `;
 
-export const ColoredIcon = styled.div`
```

Comment:
this was a workaround implemented by Vlad I think, to bypass the default fill being prioritized over provided color